### PR TITLE
Blog app

### PR DESCRIPTION
**Pull request by xrwang**
Blog app from the Getting started on Rails exercise. I have to finish the template still !

### CODE REVIEW

**Comment by tgaff** [comment]
Hi @xrwang I'm taking a look at this and seeing a bunch of files from tmp/ and log/ in the PR.  Those files are usually automatically excluded by rails when you run `rails new`.  My best guess is that you did a `git add *` instead of `git add .` maybe I'm not sure...   
Would you mind removing them though?  To do that, go into the directory for each rails app (for example `cd XiaoweiWang-xrwang/hw9/blog/` and then do:
`git rm log/*`
`git rm tmp/*`
`git commit -m "removing temp files"`

Please do that for each rails app you have.  Thanks and sorry if this was a bit confusing.

File: XiaoweiWang-xrwang/hw9/blog/db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20150910005223) do
+
```
Comment (tgaff) [inline XiaoweiWang-xrwang/hw9/blog/db/schema.rb]: This looks good for articles.  But I think maybe you didn't finish the guide yet?  
I expected to find the Comments table defined here.  I don't see the model for it either.  

Could you take a look at the getting started guide - comments starts here:
http://guides.rubyonrails.org/getting_started.html#adding-a-second-model

**Comment by tgaff** [comment]
If you get stuck removing the files we can help!  And don't forget you can use your group as a resource too; you're all doing the same HW.  :-)

**Comment by xrwang** [comment]
@tgaff updated...it looks like the tmp and log files are out of there? let me know if not...

**Comment by tgaff** [comment]
@xrwang Good work!  They're gone!  And I see bootstrap!  

thanks!

File: XiaoweiWang-xrwang/hw9/blog/app/models/article.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class Article < ActiveRecord::Base
```
Comment (tgaff) [inline XiaoweiWang-xrwang/hw9/blog/app/models/article.rb]: if you run annotate it'll add the column names as a comment here

File: XiaoweiWang-xrwang/hw9/blog/db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    t.string   "title"
+    t.text     "text"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "comments", force: :cascade do |t|
+    t.string   "commenter"
+    t.text     "body"
+    t.integer  "article_id"
```
Comment (tgaff) [inline XiaoweiWang-xrwang/hw9/blog/db/schema.rb]: does this `article_id` make sense here?  It's related to the schema stuff we're talking about today.

**Comment by tgaff** [comment]
Looks great @xrwang !  I'm glad you got bootstrap working.  

here's some more bootstrap info if you want to go a little further:
http://getbootstrap.com/components/

Did everything in the guide make sense?  Do you have any questions?

**Pull request merged**